src/org/javarosa/xform/parse/FormInstanceParser.java

```diff
@@ -0,0 +1,629 @@
+package org.javarosa.xform.parse;
+
+import org.javarosa.core.model.*;
```

Comment:
I think for new files it makes sense to follow Google's [Java style guide](https://google.github.io/styleguide/javaguide.html) (no wildcard imports).